saltpylint/thirdparty.py

```diff
@@ -83,46 +80,37 @@ def __init__(self, linter=None):
         self._inside_try_except = False
         self._inside_funcdef = False
         self._inside_if = False
-        self.cwd = self.allowed_3rd_party_modules = None
+        self.cwd = self.allowed_3rd_party_modules = []
```

Comment:
```suggestion
        self.cwd = None
        self.allowed_3rd_party_modules = []
```
What was done before was assign both variables to the same list.